src/Cake.Common/Tools/OctopusDeploy/CreateReleaseSettings.cs

```diff
@@ -46,5 +46,15 @@ public sealed class CreateReleaseSettings : OctopusDeploySettings
         /// Gets or sets a value indicating whether the Ignore Existing flag.
         /// </summary>
         public bool IgnoreExisting { get; set; }
+
+        /// <summary>
+        /// Gets or sets the octopuschannel for the new release.
+        /// </summary>
+        public string Channel { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether channel rules should be ignored
```

Comment:
i didn't find any tests regarding octopus settings or argumentbuilder, so was not able to add this